Update styles on news body width, cards on different screens

JMStewart00 opened the pull request:
## Summary / Approach
<!-- Include a summary of your changes that expands upon the title. -->
Updated the news main body content to be centered with a width of 940px + 10% of screen to account for the padding. 
Added some UX to the cards in a scrollable horizontal div to help with screen height and the shadows should help with UX and indication there is a scrollable div. 

## Testing Instruction(s)
<!-- Include a summary of how your changes should be tested, including any necessary links to the env used. -->
- Check out homepage on mobile
- Go to news article on mobile and desktop to see changes. 

## Screenshots
<!-- If appropriate include necessary screenshots to show the feature on mobile and desktop views. -->
<img width="1728" alt="Screenshot 2023-06-30 at 10 31 01 AM" src="https://github.com/lfucg/lexingtonky.gov/assets/27921423/f49f9421-167c-4da8-90c6-ba821074e9b9">
<img width="354" alt="Screenshot 2023-06-30 at 10 31 13 AM" src="https://github.com/lfucg/lexingtonky.gov/assets/27921423/dd5e2bdd-3f8f-4041-8119-aec82cf35d24">

## Metadata
<!-- Please fill out ALL metadata. Use N/A when necessary. -->
| Question | Answer |
|----------|--------|
| Did you perform a self-review of this PR? | yep
| Documentation reflects changes? | n/a
| Unit/Functional tests reflect changes? |n/a
| Did you perform browser testing? | yeah, chrome FF
| Did you provide detail in the summary on how and where to test this branch? | yep
| Relevant links | https://apaxsoftware.atlassian.net/browse/LG-106

TaylorAkin commented:
[![Visit Site](https://raw.githubusercontent.com/pantheon-systems/terminus-build-tools-plugin/master/assets/images/visit-site-36.png)](https://pr-401-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-401](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-401) for lexky-d8.

TaylorAkin commented:
@JMStewart00 We just need this to not effect the routes/pages for the menu system

https://pr-401-lexky-d8.pantheonsite.io/browse/government

TaylorAkin commented:
@JMStewart00 Same thing for /news

https://pr-401-lexky-d8.pantheonsite.io/news

JMStewart00 commented:
@TaylorAkin - Yup! Good call. That's why I wanted you to check things out. I added a parent selector to the overall width stuff to make sure that we're only grabbing the news article pages. Updated.

TaylorAkin reviewed (approved):
I made a couple of small tweaks to account for how it was effecting the front page news cards at smaller breakpoints

JMStewart00 commented on web/themes/custom/lex/scss/_variables.scss:
```diff
@@ -1,6 +1,6 @@
 $breakpoints: (
   xs: 576px,
-  sm: 768px,
+  sm: 800px,
```
@TaylorAkin 768px is usually the accepted breakpoint width for small screens. Kinda going against the grain on this one.

JMStewart00 commented on web/themes/custom/lex/old_scss/_layout.sass:
```diff
@@ -168,7 +168,7 @@ main
   .path-node.bundle-news_article &
     @media (min-width: $screen-desk-max)
-      max-width: calc(940px + 10%)
+      max-width: calc(1000px + 10%)
```
In all reality we'd want this to stay as close to `80ch` as we could. I pulled the 940px to get as close to that as possible since 80ch was pretty tight with the sidebar. Sort of went along the lines of 80ch + sidebar + padding.

JMStewart00 reviewed (commented):
Just throwing a couple comments on.
